src/Microsoft.ML.FastTree/Dataset/RepeatIntArray.cs

```diff
@@ -206,7 +206,7 @@ public override IntArray[] Split(int[][] assignment)
             }).ToArray();
         }
 
-        public override void Sumup(SumupInputData input, FeatureHistogram histogram)
+        internal override void Sumup(SumupInputData input, FeatureHistogram histogram)
```

Comment:
>internal [](start = 8, length = 8)

No need to make it internal as class is already internal. Also, I also see other methods are public too in this class.